File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
         for (int i = 0; i < nums.length; i++)
        {
            int target = nums[i];
            for (int j = 0; j < nums.length; j++)
            {
                if (target == nums[j] && i != j) return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        size_t len = nums.size();
        vector<int> output(len, 1);
        
        for (int i = 0; i < len; i++) {
            int left = i - 1;
            int right = i + 1;

            while (left > -1) {
                output[i] *= nums[left];
                left--;
            }

            while (right < len) {
                output[i] *= nums[right];
                right++;
            }
        }

        return output;
    }
};
